Parser/take_char.kind2

```diff
@@ -0,0 +1,19 @@
+// Consumes and returns the head
+// Fail on empty input
+Parser.take_char : Parser Char {
```

Comment:
Unnecessary function

Reply:
It's equivalent to parsec's `anyChar` and I implemented the other functions using `take_char` as base. But I agree that it would probably be better to do things based on `satisfy` instead and use `satisfy Bool.true` instead of this.

Reply:
I've turned it into any_char. It has different semantics compared to satisfy, so I thought it would be interesting to have.